src/braket/circuits/ascii_circuit_diagram.py

```diff
@@ -33,16 +33,14 @@ class AsciiCircuitDiagram(CircuitDiagram):
 
     @staticmethod
     def build_diagram(circuit: cir.Circuit) -> str:
-        """
-        Build an ASCII string circuit diagram.
+        """Build an ASCII string circuit diagram.
 
         Args:
-            circuit (Circuit): Circuit for which to build a diagram.
+            circuit (cir.Circuit): Circuit for which to build a diagram.
```

Comment:
Why the `cir`?

Reply:
This just matches the type. 
`import braket.circuits.circuit as cir` is the import statement. I am not sure why this was originally chosen